packages/gatsby-cli/src/reporter/reporters/index.js

```diff
@@ -1,7 +1,12 @@
-const semver = require(`semver`)
 const { isCI } = require(`ci-info`)
 
-if (semver.satisfies(process.version, `>=8`) && !isCI) {
+let inkExists = false
+try {
+  inkExists = require.resolve(`ink`)
```

Comment:
Just wondering -- why do I see backticks used everywhere for normal strings inside requires? I followed this convention here, but I am not sure if I understand it.

Reply:
This is just convention we use here. It's much nicer to work with template literals - in case we need to use interpolation we don't have to convert from "normal strings" to template literal